Origin: EmberBSD, AI-assisted.
EmberBSD keeps one Berkeley DB, db18. These packages included
databases/db4 or db5 directly, or accepted only db2..db5:
- trustedQSL 2.8 stores its data in SQLite (find_package(SQLite3)), not
  Berkeley DB: depend on sqlite3 and drop the db5 dependency.
- opensmtpd, jigdo (gtk), libpinyin and clisp-bdb use databases/db18.
- libtabe and xcin accept db18 (patch-ae/af already use the 4.3+ stat).
- cstore accepts db18.
Not submitted upstream.

--- a/databases/clisp-bdb/Makefile
+++ b/databases/clisp-bdb/Makefile
@@ -11,10 +11,10 @@
 HOMEPAGE=	http://clisp.cons.org/
 COMMENT=	CLISP Berkeley DB module
 
-SUBST_CLASSES+=			db4-libname
-SUBST_STAGE.db4-libname=	pre-configure
-SUBST_FILES.db4-libname=	modules/berkeley-db/configure
-SUBST_SED.db4-libname=		-e "/names_next_round='db '/s/db/db4/"
+SUBST_CLASSES+=			bdb-libname
+SUBST_STAGE.bdb-libname=	pre-configure
+SUBST_FILES.bdb-libname=	modules/berkeley-db/configure
+SUBST_SED.bdb-libname=		-e "/names_next_round='db '/s/db/db18/"
 
 
 MAKE_JOBS_SAFE=	no
@@ -22,5 +22,5 @@
 .include "../../lang/clisp/module.mk"
 
 .include "../../devel/gettext-lib/buildlink3.mk"
-.include "../../databases/db4/buildlink3.mk"
+.include "../../databases/db18/buildlink3.mk"
 .include "../../mk/bsd.pkg.mk"
--- a/databases/cstore/Makefile
+++ b/databases/cstore/Makefile
@@ -12,7 +12,7 @@
 HOMEPAGE=	http://db.lcs.mit.edu/projects/cstore/
 COMMENT=	Read-optimised column store relational DBMS
 
-BDB_ACCEPTED=		db4
+BDB_ACCEPTED=		db18
 USE_LANGUAGES=		c c++
 USE_TOOLS+=		gmake perl pax
 WRKSRC=			${WRKDIR}/cstore
--- a/ham/trustedQSL/Makefile
+++ b/ham/trustedQSL/Makefile
@@ -24,8 +24,6 @@
 
 .include "../../devel/cmake/build.mk"
 # \todo Check/rationalize
-CMAKE_CONFIGURE_ARGS+=	-DBDB_PREFIX=${PREFIX}
-CMAKE_CONFIGURE_ARGS+=	-DBDB_INCLUDE_DIR=${PREFIX}/include/db5
 CMAKE_CONFIGURE_ARGS+=	-DCMAKE_CONFIGURE_BUILD_WITH_INSTALL_RPATH=TRUE
 CMAKE_CONFIGURE_ARGS+=	-DCMAKE_CONFIGURE_INSTALL_PREFIX=${PREFIX}
 CMAKE_CONFIGURE_ARGS+=	-DCMAKE_CONFIGURE_INSTALL_RPATH=${PREFIX}/lib
@@ -38,10 +36,8 @@
 	${INSTALL_DATA} LICENSE.txt ${DESTDIR}${PREFIX}/share/TrustedQSL/ \
 	)
 
-# \todo Perhaps change to lmdb, and (unlikely) consider an option.
 .include "../../devel/cmake/build.mk"
-.include "../../databases/db5/buildlink3.mk"
-#.include "../../databases/lmdb/buildlink3.mk"
+.include "../../databases/sqlite3/buildlink3.mk"
 
 .include "../../textproc/expat/buildlink3.mk"
 .include "../../security/openssl/buildlink3.mk"
--- a/inputmethod/libtabe/Makefile
+++ b/inputmethod/libtabe/Makefile
@@ -12,7 +12,7 @@
 
 MAKE_JOBS_SAFE=		no
 
-BDB_ACCEPTED=		db2 db3 db4 db5
+BDB_ACCEPTED=		db18
 USE_LIBTOOL=		yes
 GNU_CONFIGURE=		yes
 CONFIGURE_ARGS+=	--with-db-inc=${BDBBASE}/include/${BDB_TYPE}
--- a/inputmethod/xcin/Makefile
+++ b/inputmethod/xcin/Makefile
@@ -69,7 +69,7 @@
 
 BUILDLINK_DEPMETHOD.libXt?=	build
 
-BDB_ACCEPTED=		db2 db3 db4 db5
+BDB_ACCEPTED=		db18
 BUILDLINK_TRANSFORM+=	l:db:${BDB_TYPE}
 
 .include "../../converters/libiconv/buildlink3.mk"
--- a/mail/opensmtpd/Makefile
+++ b/mail/opensmtpd/Makefile
@@ -120,7 +120,7 @@
 .if ${OPSYS} == "Linux"
 .  include "../../devel/libbsd/buildlink3.mk"
 .endif
-.include "../../databases/db5/buildlink3.mk"
+.include "../../databases/db18/buildlink3.mk"
 .include "../../devel/libevent/buildlink3.mk"
 .include "../../devel/zlib/buildlink3.mk"
 .include "../../net/libasr/buildlink3.mk"
--- a/net/jigdo/options.mk
+++ b/net/jigdo/options.mk
@@ -9,7 +9,7 @@
 PLIST_VARS+=		gtk
 
 .if !empty(PKG_OPTIONS:Mgtk)
-.  include "../../databases/db4/buildlink3.mk"
+.  include "../../databases/db18/buildlink3.mk"
 .  include "../../www/libwww/buildlink3.mk"
 .  include "../../x11/gtk2/buildlink3.mk"
 CONFIGURE_ARGS=		--with-gui
--- a/textproc/libpinyin/Makefile
+++ b/textproc/libpinyin/Makefile
@@ -27,7 +27,7 @@
 BUILDLINK_TRANSFORM.${os}+=	rm:-Wl,--version-script=./libzhuyin.ver
 .endfor
 
-.include "../../databases/db4/buildlink3.mk"
-BUILDLINK_TRANSFORM+=	l:db:db4
+.include "../../databases/db18/buildlink3.mk"
+BUILDLINK_TRANSFORM+=	l:db:db18
 .include "../../devel/glib2/buildlink3.mk"
 .include "../../mk/bsd.pkg.mk"
--- a/textproc/libpinyin/buildlink3.mk
+++ b/textproc/libpinyin/buildlink3.mk
@@ -11,7 +11,7 @@
 BUILDLINK_PKGSRCDIR.libpinyin?=		../../textproc/libpinyin
 
 .include "../../devel/glib2/buildlink3.mk"
-.include "../../databases/db4/buildlink3.mk"
+.include "../../databases/db18/buildlink3.mk"
 .endif	# LIBPINYIN_BUILDLINK3_MK
 
 BUILDLINK_TREE+=	-libpinyin
